inference-engine/src/mkldnn_plugin/nodes/mkldnn_matmul_node.cpp

```diff
@@ -90,14 +89,19 @@ void MKLDNNMatMulNode::setPostOps(mkldnn::primitive_attr &attr, bool initWeights
 }
 
 
-MKLDNNNode::AttrPtr MKLDNNMatMulNode::initPrimitiveAttr() const {
+MKLDNNNode::AttrPtr MKLDNNMatMulNode::initPrimitiveAttr(const VectorDims &dims) const {
     auto attr = std::make_shared<mkldnn::primitive_attr>(mkldnn::primitive_attr());
 
-    setPostOps(*attr, true);
+    setPostOps(*attr, dims, true);
 
     return attr;
 }
 
+MKLDNNNode::AttrPtr MKLDNNMatMulNode::initPrimitiveAttr() const {
+    auto dummyShape = MemoryDescUtils::makeDummyShape(getInputShapeAtPort(0));
```

Comment:
Why input shape is used but not output one?

Reply:
That's my mistake. Good catch. Thanks a lot!

Reply:
Fixed